tests/foreman/ui/test_dashboard.py

```diff
@@ -180,13 +182,22 @@ def test_positive_task_status(session):
         session.dashboard.action({
             'TaskStatus': {'state': 'running', 'result': 'pending'}
         })
-        tasks = session.task.read_all()
-        assert tasks['searchbox'] == 'state=running&result=pending'
+        searchbox = session.task.read_widget('searchbox')
+        assert searchbox == 'state=running&result=pending'
+        session.task.set_chart_filter('RunningChart')
+        row_count = session.task.table_row_count()
+        chart = session.task.read_widget('RunningChart')
+        assert row_count == int(chart['total']['Total'])
         session.dashboard.action({
```

Comment:
Do you think we need to add one more assertion to see that we are navigated to latest result of task status? There could be a possibility that we could have multiple entries of 'TaskStatus': {'state': 'running', 'result': 'pending'}(Inregression). We should make sure that the we are navigated the to the task that we created as part of the test

Reply:
If I understand correctly, there is still only one row per 'TaskStatus': {'state': 'running', 'result': 'pending'} in 'Task Status' table in dashboard. So clicking in table gets you to the list of tasks. There is no change of multiple entries.

Reply:
Yes if we are running tests in threads there could be a similar task is running and pending status?

Reply:
Yes, there could be a similar task, but this part of test validate just filtering and if number shown in Tasks dashboard is same as number of tasks in table. Maybe I am missing something, but you don't care if there are more similar tasks.